src/QMCApp/QMCMain.cpp

```diff
@@ -124,18 +124,17 @@ QMCMain::QMCMain(Communicate* c)
 
   // Record features configured in cmake or selected via command-line arguments to the printout
   app_summary() << std::endl;
-#if !defined(ENABLE_OFFLOAD) && !defined(ENABLE_CUDA) && !defined(QMC_CUDA) && !defined(ENABLE_ROCM)
+#if !defined(ENABLE_OFFLOAD) && !defined(ENABLE_CUDA) && !defined(QMC_CUDA) && !defined(ENABLE_HIP)
   app_summary() << "  CPU only build" << std::endl;
 #else
 #if defined(ENABLE_OFFLOAD)
   app_summary() << "  OpenMP target offload to accelerators build option is enabled" << std::endl;
 #endif
-#if defined(ENABLE_CUDA) || defined(QMC_CUDA)
+#if defined(ENABLE_HIP) || defined(QMC_CUDA2HIP)
```

Comment:
This is probably out of scope for this PR.

We have too many hip options.  If ENABLE_HIP is on QMC_CUDA2HIP must be on since you are by definition on a HIP system and if the CUDA is built it will be broken.
I don't see any good reason to allow QMC_CUDA2HIP and not ENABLE_HIP.

However as long as all those options exist and can be used in different combination they should probably give separate output.  Otherwise you need to have access to the build options to really know what you are running anyway.

Reply:
That is not true. ENABLE_HIP is direct HIP source code in AFQMC.

Reply:
Updated printout.

Reply:
@PDoakORNL how about the current printout?